src/Plugin/KeyInput/ApigeeAuthKeyInput.php

```diff
@@ -89,6 +89,9 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
         'required' => [$state_for_public, $state_for_private],
       ],
     ];
+    if (EdgeKeyTypeInterface::INSTANCE_TYPE_PRIVATE === $values['instance_type']) {
```

Comment:
@phdhiren I think we should check `!empty` for `instance_type` here also.

Reply:
@shishir-intelli, thanks. Now this also covered